### PR TITLE
Templates style markup

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@Mitso why was that test removed?

**Comment by codiebeulaine** [comment]
👍

File: molo/yourtips/models.py [hunk 1]
```diff
@@ -75,11 +75,25 @@ class YourTip(TranslatablePageMixinNotRoutable, Page):
...
         null=True, blank=True,
+        verbose_name="Homepage Banner Copy",
         max_length=255)
 
     def get_effective_extra_style_hints(self):
             return self.extra_style_hints
 
+    @staticmethod
+    def get_number_of_tips():
+        print YourTipsArticlePage.objects.all(
```
Comment (smarzban) [inline molo/yourtips/models.py:86]: @cblignaut @Mitso print statement

File: CHANGES.rst [hunk 2]
```diff
@@ -4,3 +4,8 @@ CHANGE LOG
 0.0.1
 -----
 - initial release
+
+1.0.0
```
Comment (smarzban) [inline CHANGES.rst:8]: @Mitso @cblignaut we usually add the change logs in the release process, it's ok now but in feature lets update the change log in the release branch

File: VERSION [hunk 3]
```diff
@@ -1 +1 @@
-0.0.1
+1.0.0
```
Comment (smarzban) [inline VERSION:1]: @Mitso @cblignaut same here it's better to do this in the release branch

File: molo/yourtips/migrations/0005_merge.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```
Comment (smarzban) [inline molo/yourtips/migrations/0005_merge.py:1]: @cblignaut what's this migration for?

File: molo/yourtips/migrations/0003_auto_20170705_1011.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```
Comment (smarzban) [inline molo/yourtips/migrations/0003_auto_20170705_1011.py:1]: @cblignaut lets look at the migrations again, I think we need to do some cleanup

**Pull request merged**